docs/01-marketing/05-user-stories.md

```diff
@@ -0,0 +1,24 @@
+# User Stories
+
+## 1. Finding units of measurement for a user task
+
+**As** a user of the service, \
+**I want** to see possible units for my task, \
+**To be able** to choose the most suitable options.
```

Comment:
This is very smooth story.
The aim of user stories is specifying user requirements. You should imagine user tasks.
For example, for marketplace we need units to ...
1. provide technical parameters of the products
2. compare different products specified with parameters in different units

Reply:
Left as is. I want to focus on other tasks. I'll fix it later if I have time.